### ISSUE TITLE
Plotting result example

### ISSUE THREAD

**Issue by archetype-2** [SerdarHelli/Segmentation-of-Teeth-in-Panoramic-X-ray-Image#6]
hi there,

getting very bad results with the provided code and given sample data, even though training accuracy is good. pictured below are the errors. not sure whats going on, please help! ;(

![2d](https://user-images.githubusercontent.com/110601496/220262069-ceab1800-8321-4140-a050-8656426c8712.png)

![2d2](https://user-images.githubusercontent.com/110601496/220262101-9b35ce99-8771-4d35-9b9e-ba1c7dfc7afb.png)

**Comment by SerdarHelli** [comment]
It shouldnt be . can you send whole code pls ?

**Comment by archetype-2** [comment]
It is your Main.ipynb code sir. Trained the model with dataset given in thr github repository.
![2d3](https://user-images.githubusercontent.com/110601496/220265636-d936c96c-6e5a-4a8e-9c3c-812ce28ad742.png)
![2d4](https://user-images.githubusercontent.com/110601496/220265669-c7e4216a-633e-4c07-a96d-760f37e906d1.png)

**Comment by SerdarHelli** [comment]
maybe you can try with 

img=cv2.drawContours(img,cnts,-1,(255,0,0),2)

**Comment by archetype-2** [comment]
I see, thank you! Now the  ##Plotting - RESULT  seems to be resolved after making the above change.
However, the ##Plotting - RESULT Example with CCA_Analysis is still giving the below result ;(
![2d5](https://user-images.githubusercontent.com/110601496/220294077-0f2e25cb-828d-4108-80b3-d63d79f19488.png)
![2d6](https://user-images.githubusercontent.com/110601496/220294122-e11db384-08be-4ff7-b704-62261193ecfd.png)

**Comment by SerdarHelli** [comment]
Sorry, I forgot to answer :D . I couldnt understand why you are getting like that ,when you got good segmentation map .Can you send email with the predicted segmentation map ?

**Comment by SerdarHelli** [comment]
Hey , Did you send ? I can examine. You can just send your prediction map

**Comment by AsmaeKarmouchi** [comment]
![code _Main](https://user-images.githubusercontent.com/122885582/235980967-d0596365-bb63-4904-8f1f-37c73a24f61c.jpeg)
please can you suggest a solution for my problem because this block cannot be executed I left it the last time for more than 72 min its execution is not finished

**Comment by SerdarHelli** [comment]
> ![code _Main](https://user-images.githubusercontent.com/122885582/235980967-d0596365-bb63-4904-8f1f-37c73a24f61c.jpeg) please can you suggest a solution for my problem because this block cannot be executed I left it the last time for more than 72 min its execution is not finished

Maybe you should look at your env. Did you install opencv properly ?

**Comment by AsmaeKarmouchi** [comment]
thank you for your answer , opencv already installed correctly, I am talking about the blocks of two functions model.compile and model.fit this block fails to complete its execution.
![opencv](https://user-images.githubusercontent.com/122885582/236046996-e61929fd-2e0d-44b5-a230-890bdda1f731.jpeg)

**Comment by SerdarHelli** [comment]
I have no idea . Maybe it is realated your compute power. It should give error , did you check out it under tensorflow and keras issues
